sql/core/src/main/java/org/apache/spark/sql/sources/v2/Table.java

```diff
@@ -27,8 +27,8 @@
  * implementation can be a directory on the file system, a topic of Kafka, or a table in the
  * catalog, etc.
  * <p>
- * This interface can mixin the following interfaces to support different operations, like
- * {@code SupportsRead}.
+ * This interface can mixin {@link SupportsRead} and {@link SupportsWrite} to provide data reading
```

Comment:
now `Table` only has 2 mixin traits, we should list all of them instead of saying `... like SupportsRead`.

Reply:
This is going to cause a conflict with #24246, which moves this class into catalyst. The links will no longer work after that move, so I'd rather avoid the conflict and make this change in that PR. That also make this PR touch one fewer file, so that's good. Could you remove this change?